packages/perseus/src/widgets/__tests__/interactive-graph.test.ts

```diff
@@ -314,4 +315,112 @@ describe("locked layer", () => {
         expect(points[0]).toHaveStyle({fill: color.green, stroke: color.green});
         expect(points[1]).toHaveStyle({fill: color.green, stroke: color.green});
     });
+
+    test("should render locked lines", () => {
+        // Arrange
+        const {container} = renderQuestion(segmentWithLockedLineQuestion, {
+            flags: {
+                mafs: {
+                    segment: true,
+                },
+            },
+        });
+
+        // Act
+        // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+        const lines = container.querySelectorAll(".locked-line");
+
+        // Assert
+        expect(lines).toHaveLength(2);
+    });
+
+    test("should render locked lines with styles", () => {
+        // Arrange
+        const {container} = renderQuestion(segmentWithLockedLineQuestion, {
+            flags: {
+                mafs: {
+                    segment: true,
+                },
+            },
+        });
+
+        // Act
+        // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+        const lines = container.querySelectorAll(".locked-line line");
+
+        // Assert
+        expect(lines).toHaveLength(2);
+        expect(lines[0]).toHaveStyle({stroke: color.purple});
+        expect(lines[1]).toHaveStyle({stroke: color.green});
+    });
+
+    test("should render locked lines with shown points", async () => {
+        // Arrange
+        const {container} = renderQuestion(segmentWithLockedLineQuestion, {
+            flags: {
+                mafs: {
+                    segment: true,
+                },
+            },
+        });
+
+        // Act
+        // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+        const points = container.querySelectorAll(".locked-line circle");
+
+        // Assert
+        expect(points).toHaveLength(4);
+        // Two points for each line
+        expect(points[0]).toHaveStyle({
+            fill: color.purple,
+            stroke: color.purple,
+        });
+        expect(points[1]).toHaveStyle({
+            fill: color.white,
+            stroke: color.purple,
+        });
+        expect(points[2]).toHaveStyle({
+            fill: color.white,
+            stroke: color.green,
+        });
+        expect(points[3]).toHaveStyle({fill: color.green, stroke: color.green});
```

Comment:
NIT: Should this have the same line breaks as the other `expect` statements before it?

Reply:
Oh yeah, I think that fix is in a follow-up PR (the MathJax colors one), so I'll leave it like this here for now